site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -6,25 +6,79 @@ updated: 2018-02-25
 description: An overview of the purpose of Chrome Extensions and how they're developed.
 ---
 
-Extensions are small software programs that customize the browsing experience. They enable users to
-tailor Chrome functionality and behavior to individual needs or preferences. They are built on web
-technologies such as HTML, JavaScript, and CSS.
+This page provides a very brief introduction to what extensions are and how you can make a trivial one.
 
-An extension must fulfill a [single purpose][1] that is narrowly defined and easy to understand. A
+
+## About extensions
+
+Extensions are small software programs that customize the browsing experience. They let users tailor
+Chrome functionality and behavior in many ways, providing things like:
+
+* Productivity tools
+* Web page content enrichment
+* Information aggregation
+* Fun and games
+
+These are just a few examples of the many things that extensions can do. See the [Chrome Web
+Store][5] to see thousands of different examples of published extensions.
+
+### How do extensions work?
+
+Extensions are built on web technologies such as HTML, JavaScript, and CSS.  They run in a separate,
+sandboxed execution environment and interact with the Chrome browser.  Consider the context for this
+-- Chrome runs on a device, connecting the user to resources on the Web, as depicted in this context
+diagram:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/5qVlqGn6vVMCWCSRclOY.png", alt="Context diagram of Chrome with an extension", width="517", height="476" %}
+
+Also shown in the diagram is an extension "plugged into" the Chrome browser. The extension can
+access and modify selected resources and actions in Chrome. There are two aspects to consider:
+
+**The extensions mechanism**&emsp;This provides a consistent way for users to install and activate
```

Comment:
I don't love the phrasing of this as "the extensions mechanism" - it's not clear to me what this is referring to, or what the bounds of it are.  If this is just the installation and activation, it's more the UI than the "mechanism".  Can you add a bit more detail here about what this is trying to get at?

Reply:
Done: Changed it to "extensions user interface" and made it about "manage their extensions"